content/en/blog/_posts/2024-05-13-gateway-api-v1.1/index.md

```diff
@@ -0,0 +1,267 @@
+---
+layout: blog
+title: "Gateway API v1.1: Feature Graduation and Other Updates"
```

Comment:
Completely skipped over this section in the initial review, but maybe we should include some of the headline features in the title. Also considered session persistence of client cert verification but either of those lead to a pretty long title.

```suggestion
title: "Gateway API v1.1: Service Mesh, GRPCRoute, and a whole lot more"
```